README.md

```diff
@@ -25,6 +25,61 @@ Run it:
   $ go run server.go -config=/path/to/settings.yaml
   ```
 
+## Development Environment
+
+### Install Go
+Follow the instructions to [install Go](https://golang.org/doc/install).
+
+### Install the React development environment
+#### Pre-requisites
+- Install the build packages
+```bash
+sudo apt-get install build-essential libssl-dev
+```
+
+- Install NVM
+Install the [Node Version Manager](https://github.com/creationix/nvm) that will allow a specific
+version of Node.js to be installed. Follow the installation instructions.
+
+- Install the latest stable Node.js and npm
+The latest stable (LTS) version of Node can be found on the [Node website](nodejs.org).
+```bash
+# Overview of available commands
+nvm help
+
+# Install the latest stable version
+nvm install v4.4.3
+
+# Select the version to use
+nvm ls
+nvm use v4.4.3
+
+# Install gulp globally
+npm install -g gulp
+```
+
+- Install the nodejs dependencies
+```bash
+cd identity-vault
+npm install
+```
+
+### Working with React
+
+#### Build the project bundle
+```bash
+# Select the version to use
+nvm ls
+nvm use v4.4.3
+gulp
```

Comment:
I have tried following these new instructions to set up a dev environment, and when I arrive to this point I get the following error:

ara@sushirirder:~/.go/src/github.com/ubuntu-core/identity-vault$ gulp
[12:56:45] Using gulpfile ~/.go/src/github.com/ubuntu-core/identity-vault/gulpfile.js
[12:56:45] Starting 'compile_components'...
[12:56:45] Starting 'compile_models'...
[12:56:45] Starting 'sass'...
[12:56:45] Finished 'sass' after 5.13 ms
[12:56:48] Finished 'compile_models' after 2.87 s
[12:56:48] Finished 'compile_components' after 3.04 s
[12:56:48] Starting 'compile_app'...
[12:56:48] Finished 'compile_app' after 52 ms
[12:56:48] Starting 'build'...

events.js:141
      throw er; // Unhandled 'error' event
      ^
Error: module "react-intl/lib/locale-data/en" not found from "/home/ara/.go/src/github.com/ubuntu-core/identity-vault/webapp/build/fake_28e71269.js"
    at notFound (/home/ara/.go/src/github.com/ubuntu-core/identity-vault/node_modules/gulp-browserify/node_modules/browserify/index.js:803:15)
    at /home/ara/.go/src/github.com/ubuntu-core/identity-vault/node_modules/gulp-browserify/node_modules/browserify/index.js:754:23
    at /home/ara/.go/src/github.com/ubuntu-core/identity-vault/node_modules/gulp-browserify/node_modules/browserify/node_modules/browser-resolve/index.js:185:24
    at /home/ara/.go/src/github.com/ubuntu-core/identity-vault/node_modules/gulp-browserify/node_modules/browserify/node_modules/resolve/lib/async.js:44:14
    at process (/home/ara/.go/src/github.com/ubuntu-core/identity-vault/node_modules/gulp-browserify/node_modules/browserify/node_modules/resolve/lib/async.js:113:43)
    at /home/ara/.go/src/github.com/ubuntu-core/identity-vault/node_modules/gulp-browserify/node_modules/browserify/node_modules/resolve/lib/async.js:122:21
    at load (/home/ara/.go/src/github.com/ubuntu-core/identity-vault/node_modules/gulp-browserify/node_modules/browserify/node_modules/resolve/lib/async.js:54:43)
    at /home/ara/.go/src/github.com/ubuntu-core/identity-vault/node_modules/gulp-browserify/node_modules/browserify/node_modules/resolve/lib/async.js:60:22
    at /home/ara/.go/src/github.com/ubuntu-core/identity-vault/node_modules/gulp-browserify/node_modules/browserify/node_modules/resolve/lib/async.js:16:47
    at FSReqWrap.oncomplete (fs.js:82:15)